### PR TITLE
add name fields to commands

### PR DESCRIPTION
**Pull request by Pnom**

### CODE REVIEW

File: src/analog/commands/uplink/getChannelsStatus.ts [hunk 1]
```diff
@@ -44,6 +45,7 @@ interface ITemperatureSensorStatus {
 
 interface IChannelStatus {
     type: types.TUint8;
+    typeName: string,
```
Comment (DarkPark) [inline src/analog/commands/uplink/getChannelsStatus.ts:48]:
```suggestion
    typeName?: string,
```

File: src/analog/commands/uplink/getChannelsTypes.ts [hunk 2]
```diff
@@ -64,4 +77,7 @@ export const fromBytes = ( data: types.TBytes ): IChannelsMap => ({channelsMap:
  * @param parameters - command payload
  * @returns full message (header with body)
  */
-export const toBytes = ( {channelsMap}: IChannelsMap ): types.TBytes => (command.toBytes(id, channelsMap));
+export const toBytes = ( {channels}: IChannels ): types.TBytes => (command.toBytes(
```
Comment (DarkPark) [inline src/analog/commands/uplink/getChannelsTypes.ts:80]: I would prefer this way:

```js
export const toBytes = ( {channels}: IChannels ): types.TBytes => (
    command.toBytes(
        id,
        channels.map(channel => channel.type)
    )
);
```

this is even better (as code is rather simple and to need to split to many lines):

```js
export const toBytes = ( {channels}: IChannels ): types.TBytes => (
    command.toBytes(id, channels.map(channel => channel.type))
);
```

**Review by DeadByElpy — DISMISSED** [review]

File: src/analog/commands/uplink/getChannelsTypes.ts (hunk 2 above)
Comment (Pnom) [reply src/analog/commands/uplink/getChannelsTypes.ts]: fixed

File: src/analog/commands/uplink/getChannelsTypes.ts [hunk 3]
```diff
@@ -22,25 +22,36 @@
...
 export const id: types.TCommandId = 0x331f;
 export const name: types.TCommandName = 'getChannelsTypes';
 export const headerSize = 3;
 
 
-interface IChannelsMap {
-    channelsMap: Array<types.TUint8>
+interface IChannel {
+    type: types.TUint8,
+    name?: string
```
Comment (DarkPark) [inline src/analog/commands/uplink/getChannelsTypes.ts:35]:
```suggestion
    typeName?: string
```

like in IChannelStatus?

File: src/analog/constants/channelsNames.ts [hunk 4]
```diff
@@ -0,0 +1,5 @@
+import * as channelsTypes from './channelsTypes.js';
```
Comment (DarkPark) [inline src/analog/constants/channelsNames.ts:1]: would be nice to add all these new const files to src/analog/constants/index.ts

File: src/mtx1/utils/CommandBinaryBuffer.ts [hunk 5]
```diff
@@ -801,6 +805,7 @@ export interface IEvent {
     minutes: types.TUint8,
     seconds: types.TUint8,
     event: types.TUint8,
+    name?: string,
```
Comment (DarkPark) [inline src/mtx1/utils/CommandBinaryBuffer.ts:808]:
```suggestion
    eventName?: string,
```

?

File: src/mtx1/utils/CommandBinaryBuffer.ts [hunk 6]
```diff
@@ -1406,10 +1411,16 @@ CommandBinaryBuffer.getDefaultOperatorParameters = (): IOperatorParameters => (
...
+    const name = frameNames[type] as string;
+    const destination = this.getUint16();
+    const source = this.getUint16();
+
     return {
-        type: this.getUint8(),
-        destination: this.getUint16(),
-        source: this.getUint16()
+        type,
+        name,
```
Comment (DarkPark) [inline src/mtx1/utils/CommandBinaryBuffer.ts:1421]:
```suggestion
        typeName,
```

?

**Review by DeadByElpy — DISMISSED** [review]

File: src/mtx3/commands/uplink/getEvents.ts [hunk 7]
```diff
@@ -0,0 +1,172 @@
...
+                date: 12
+            },
+            eventsNumber: 2,
+            events: [
+                {
+                    hours: 1,
+                    minutes: 12,
+                    seconds: 33,
+                    event: 157,
+                    name: 'POWER_OVER_RELAY_OFF',
```
Comment (DarkPark) [inline src/mtx3/commands/uplink/getEvents.ts:100]:
```suggestion
                    eventName: 'POWER_OVER_RELAY_OFF',
```

?

or `event` -> `id` ?

File: src/analog/commands/uplink/getChannelsTypes.ts (hunk 3 above)
Comment (Pnom) [reply src/analog/commands/uplink/getChannelsTypes.ts]: fixed

File: src/mtx1/utils/CommandBinaryBuffer.ts (hunk 6 above)
Comment (Pnom) [reply src/mtx1/utils/CommandBinaryBuffer.ts]: fixed

File: src/mtx3/commands/uplink/getEvents.ts (hunk 7 above)
Comment (Pnom) [reply src/mtx3/commands/uplink/getEvents.ts]: fixed

File: src/analog/constants/channelsNames.ts (hunk 4 above)
Comment (Pnom) [reply src/analog/constants/channelsNames.ts]: fixed

**Pull request merged**